app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class API::V1::UsersController < API::V1::APIController
+  def create
+    @user = User.where(
+      email: params[:data][:attributes][:email]
+    ).first_or_initialize(password: SecureRandom.uuid) # tmp pw for now
```

Comment:
Can we formalize this so we skip a password for these users? Or do something more explicit to make it clear there's a user without their own generated password here? 

I'm worried that we can't easily identify users from users without account passwords they created right now (unless we already solved this and i'm just not remembering).

Reply:
How about I create a user role called "lead"? And then we can filter on those if needed.

Reply:
👍 great. As long as there's something we can work with so we know they dont really have a password later when they try to sign up.